scripts/dataset_processing/tts/create_speaker_map.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script takes a list of TTS manifests and creates a JSON mapping the input speaker names to
+unique indices for multi-speaker TTS training.
+
+To ensure that speaker names are unique across datasets, it is recommended that you prepend the speaker
+names in your manifest with the name of the dataset.
+
+$ python <nemo_root_path>/scripts/dataset_processing/tts/create_speaker_map.py \
+    --manifest_paths=manifest1.json,manifest2.json \
+    --speaker_map_path=speakers.json
+
+Example output:
+
+{
+    "vctk_p225": 0,
+    "vctk_p226": 1,
+    "vctk_p227": 2,
+    ...
+}
+
+"""
+
+import argparse
+import json
+from pathlib import Path
+
+from nemo.collections.asr.parts.utils.manifest_utils import read_manifest
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description="Create mapping from speaker names to numerical speaker indices.",
+    )
+    parser.add_argument(
+        "--manifest_paths", required=True, type=str, help="Path to training manifests, comma delimited",
```

Comment:
Could you please change this to use an honest list, not split things by a comma?
Argparse allows to do this through [actions](https://docs.python.org/3/library/argparse.html#action)

Example:
```
import argparse

if __name__ == "__main__":
    parser = argparse.ArgumentParser()
    parser.add_argument("--manifest_path", action="append") 

    args = parser.parse_args()
    print(args.manifest_path)
```
```
$ python test.py --manifest_path foo --manifest_path bar
['foo', 'bar']
```